android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -594,4 +595,11 @@ public String getApiName()
 	{
 		return "Ti.UI.Window";
 	}
+
+	private void logDeprecatedPropertiesWarning() {
```

Comment:
the properties that they should now use aka the transition properties, are not uses in the method and so it will fail and so i will leave that part out of the warning